demosplan/DemosPlanCoreBundle/DataGenerator/CustomFactory/StatementFactory.php

```diff
@@ -196,8 +194,6 @@ protected function makeStatementData(): array
      *
      * @param array $baseData
      *
-     * @return mixed
-     *
      * @throws InvalidUserDataException on invalid user value
      */
     protected function determineUserInfo($baseData)
```

Comment:
Also the parameter always have the type array as well as the return type. These can be added here.

Reply:
done in [685364f](https://github.com/demos-europe/demosplan-core/pull/2960/commits/685364f97784d85dd520137c6bcee9f67c93e07c)

Reply:
I meant within the signature of the method, whereby the docblock can be reduced.

Reply:
done in [6f3d74a](https://github.com/demos-europe/demosplan-core/pull/2960/commits/6f3d74a72deeeeba4cf93caf6dd9e632908700d0)